src/cores/events/EditForm.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import Form from './Form';
+import * as eventsServices from '../../services/events.services';
+import { editEvent } from '../../features/events/eventSlice';
+import { useState, useEffect } from "react";
+
+
+const EditForm = () => {
+  const dispatch = useDispatch();
+  const {userToken} = useSelector(state => state.auth)
+  const events = useSelector((state)=> state.events.eventsAll)
+  const { id } = useParams();
+  const [eventFound, setEventFound] = useState();
+
+  const edit = async (event, id) => {
+    try {
+      if (event.status === "Activo"){
+        event.status = true;
+      } else {
+        event.status = false;
+      }
+      await eventsServices.editEvent(userToken, event, id);
+      dispatch(editEvent(event));
+
```

Comment:
```suggestion
      const eventData = {
          ...event,
          status: !!(event.status === "Activo")
      }
      await eventsServices.editEvent(userToken, eventData, id);
      dispatch(editEvent(eventData));

```
Asi evitamos modificar el objeto original que te trae los datos